docs/pages/includes/role-spec.mdx

```diff
@@ -206,10 +206,11 @@ spec:
     kubernetes_resources:
         # The resource kind. Teleport currently supports:
         # - * (all resources)
+        # - <arbitrary string> (custom resources, e.g. 'stable.example.com/v1/myresources')
         # - pod
         # - secret
         # - configmap
-        # - namespace
+        # - namespace (including all resources kinds in the namespace)
```

Comment:
Is this really the case? Having the same name as an existing resource is pretty confusing, and makes the functionality non-obvious.

Reply:
Agreed, but that how it works today. Adding the details to be explicit as it is indeed non-obvious.